app/players/elves/elf.py

```diff
@@ -0,0 +1,11 @@
+from app.players.player import Player
+
+
+class Elf(Player):
+    def __init__(self, nickname: str, musical_instrument: str) -> None:
+        super().__init__(nickname)
+        self._musical_instrument = musical_instrument
+
+    def play_elf_song(self) -> None:
+        print(f"{self.nickname} is playing a song on the "

```

Comment:
After opening the parentheses, write each the arguments on a new line, including the first one, the closing bracket is also written on a new line